psvpfsparser/Utils.cpp

```diff
@@ -65,35 +65,28 @@ void getFileListNoPfs(boost::filesystem::path root_path, std::set<boost::filesys
          const boost::filesystem::path cp = (*i);
 
          //skip paths that are not included in files.db
+         //i.nopush(true) will skip recursion into directory
 
          //skip pfs directory
-         if(boost::starts_with(cp, (root_path / boost::filesystem::path("sce_pfs"))))
+         if(cp.filename() == boost::filesystem::path("sce_pfs")) {
+            i.no_push(true);
             continue;
+         }
 
          //skip packages
-         if(boost::starts_with(cp, (root_path / boost::filesystem::path("sce_sys") / boost::filesystem::path("package"))))
-            continue;
-
-         //skip pfs inside sce_pfs (for ADDCONT)
-         if(boost::ends_with(cp, boost::filesystem::path("sce_pfs")))
-            continue;
-
-         //skip pfs inside sce_pfs (for ADDCONT)
-         if(boost::ends_with(cp, boost::filesystem::path("sce_pfs") / boost::filesystem::path("files.db")))
-            continue;
-
-         //skip pfs inside sce_pfs (for ADDCONT)
-         if(boost::ends_with(cp, boost::filesystem::path("sce_pfs") / boost::filesystem::path("unicv.db")))
+         if(cp == (root_path / boost::filesystem::path("sce_sys/package"))) {
```

Comment:
can we use / operator instead of / in string constant?

Reply:
fixed

https://github.com/motoharu-gosuto/psvpfstools/blob/94471aefbc4e991bc2981f1489237df9c7ff44f2/psvpfsparser/Utils.cpp#L77

Reply:
perfect.